azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -292,7 +299,7 @@ private void attachKafkaAppender(KafkaLog4jAppender appender) {
 
   private KafkaLog4jAppender createKafkaAppender() throws UndefinedPropertyException {
     KafkaLog4jAppender kafkaProducer = new KafkaLog4jAppender();
-    kafkaProducer.setSyncSend(false);
+    kafkaProducer.setSyncSend(true);
```

Comment:
I noticed messages were being sent out of order. Turning on sync send fixes this.